src/stylesheets/project_specific/blog/blog.scss

```diff
@@ -1,6 +1,4 @@
 $section-height: 550px;
```

Comment:
It will be probably better to rename this var to specific thing (ex.map gallery height) and make it to local to that file. I will do that in separate PR